### PR TITLE
A module for displaying NVIDIA GPUs' temperature

### PR DESCRIPTION
**Pull request by jmdana**

### CODE REVIEW

File: py3status/modules/nvidia_temp.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
```
Comment (ultrabug) [inline py3status/modules/nvidia_temp.py]: please, don't add a python shebang and make sure your encoding line is `# -*- coding: utf-8 -*-` exactly (QA)

File: py3status/modules/nvidia_temp.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+"""
+Module for displaying NVIDIA GPU temperature
+
+Requires:
+    - nvidia-smi
+
+@author jmdana <https://github.com/jmdana>
+@license GPLv3 <http://www.gnu.org/licenses/gpl-3.0.txt>
```
Comment (ultrabug) [inline py3status/modules/nvidia_temp.py]: Would you agree to license using BSD instead ? I try to lower the licenses mess in py3status as much as possible.

File: py3status/modules/nvidia_temp.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+Module for displaying NVIDIA GPU temperature
+
```
Comment (ultrabug) [inline py3status/modules/nvidia_temp.py]: Here we miss a documentation about configuration parameters please (see the example module)

File: py3status/modules/nvidia_temp.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                format_prefix=self.format_prefix
+            )
+
+            color = self.color_bad or i3s_config['color_bad']
+
+        response = {
+            'cached_until': time() + self.cache_timeout,
+            'full_text': output,
+            'color': color,
+        }
```
Comment (ultrabug) [inline py3status/modules/nvidia_temp.py]: please order dict keys alphabetically (QA)

**Comment by ultrabug** [comment]
Thanks a lot for yet another contribution @jmdana :) I made a few comments plz.

**Comment by jmdana** [comment]
Done. Do you have a place where all the coding style can be found?

**Comment by ultrabug** [comment]
Cool, looking good mate. Last question, did you check that it works with both python2 and python3 ?

I'll make a contributors wiki page asap..

**Comment by jmdana** [comment]
Yes, it does work for me.

**Comment by ultrabug** [comment]
Thanks a lot Jose !

**Pull request merged**